service/models.py

```diff
@@ -194,3 +194,13 @@ def find_by_name(cls, name):
         """
         logger.info("Processing name query for %s ...", name)
         return cls.query.filter(cls.name == name)
+
+    @classmethod
+    def find_by_status(cls, status):
+        """Returns all Promotions with the given status
```

Comment:
"""Returns all Promotions by their status
:param status: True for promotions that are active
:type available: bool
:return: a collection of Promotions that are active or inactive based on given status
:rtype: list
"""